cmd/repo-updater/repoupdater/server_test.go

```diff
@@ -774,34 +654,33 @@ func TestRepoLookup_syncer(t *testing.T) {
 			HTTPCloneURL: "https://git-codecommit.us-west-1.amazonaws.com/v1/repos/stripe-go",
 			LastModified: &now,
 		},
-	})
-
-	s := Server{
-		Syncer:      &repos.Syncer{},
-		Store:       store,
-		InternalAPI: &internalAPIFake{},
 	}
 
-	t.Run("not found", func(t *testing.T) {
-		have, err := s.repoLookup(ctx, protocol.RepoLookupArgs{Repo: "github.com/a/b"})
-		if err != nil {
-			t.Fatal(err)
-		}
-		want := &protocol.RepoLookupResult{ErrorNotFound: true}
-		if !reflect.DeepEqual(have, want) {
-			t.Error(cmp.Diff(have, want))
-		}
-	})
-
 	testCases := []struct {
-		name         string
-		repoName     string
-		wantRepoInfo *protocol.RepoInfo
+		name                  string
+		args                  protocol.RepoLookupArgs
+		reposInStore          []*repos.Repo
+		want                  *protocol.RepoLookupResult
+		sourcegraphDotComMode bool
+		githubDotComSource    *internalGithubDotComSourceFake
+		storeAssertion        func(context.Context, *testing.T, *repos.FakeStore)
```

Comment:
Do these assertion vary enough that they need to be modeled as a function? If it's always asserting the same thing then it doesn't justify the additional complexity. We only have these assertion style function in other places because we have a bunch of different assertion functions we use.

So here, it'd be simpler to have:

```suggestion
		after        []*repos.Repo
```